libs/application/templates/health-insurance/src/forms/messages.ts

```diff
@@ -472,30 +492,31 @@ export const m = defineMessages({
   },
   alreadyInsuredTitle: {
     id: 'hi.application:alreadyInsured.title',
-    defaultMessage: 'Already insured',
+    defaultMessage: 'Þú ert nú þegar sjúkratryggður',
     description: 'Already insured',
   },
   alreadyInsuredDescription: {
     id: 'hi.application:alreadyInsured.description',
     defaultMessage:
-      'It seems like you already have a health insurance in Iceland',
+      'Þú ert nú þegar með virka sjúkratryggingu hjá Sjúkratryggingum Íslands og þarft því ekki að sækja um.\n Nánari upplýsingar er að finna á heimasíðu Sjúkratrygginga Íslands.',
     description: 'already insured',
```

Comment:
If you want to, you can update the descritpion field to the english text. Otherwise it doesn't matter.